src/dataIntegration/metabotools/prepIntegrationQuant.m

```diff
@@ -73,8 +73,6 @@ function prepIntegrationQuant(model,metData,exchanges,samples,test_max,test_min,
      if ~isempty(G)
       Secretion_not_possible(l,1)= FBA_all_secreted_names{i};
       l=l+1;
-     else
-         Secretion_not_possible = {};
```

Comment:
@maikeaurich, what will be the value of `Secretion_not_possible` when `G` is empty? May we initialize `Secretion_not_possible` somehow?